setup.py

```diff
@@ -1,12 +1,12 @@
 #!/usr/bin/python3
 
-from setuptools import setup, find_namespace_packages
+from setuptools import setup, find_packages
 
 setup(
     name="TWCManager",
     version="1.2.1",
     package_dir={"": "lib"},
-    packages=find_namespace_packages(where="lib"),
+    packages=find_packages(where="lib"),
```

Comment:
This would break the setup - as we use the namespace convention for our libraries. What made you change this out of interest?

Reply:
it didn't work, I get an error and couln't find any way to solve it but to remove the namespace:
"ImportError: cannot import name 'find_namespace_packages'"

Reply:
It probably means that you are aither running setup.py with python2 or your setuptools is too old. In that case try `pip install --upgrade setuptools` (Maybe use pip3 instead of pip)

Reply:
I already did it but it didn't work, I'll give it another try anyway

Reply:
I get the same error, I going to give up on this...

Reply:
Don't stress, it probably doesn't matter too much in your instance as it's working for you. As @tjikkun has mentioned, namespace packages were introduced in a later version, so if you see an error that says that the function can't be found, that is why. The problem is our modules are not in a traditional (non-namespace) format, so while what you changed it to might run on your older version, it would actually break newer python installs that do support namespace format because it would skip over all of the modules.

If you want to roll back the change to this file in your branch (limit_amps_from_grid) that you've submitted a PR for, you can do this:

```
git checkout v1.2.1 -- setup.py
git commit
git push
```

And the change should be reverted from your PR.